<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BRWidget</class>
 <widget class="QWidget" name="BRWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>182</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Batch record</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0">
      <item row="2" column="3">
       <widget class="QLabel" name="label_invoice">
        <property name="text">
         <string>Invoice</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="DateInput" name="edit_expiry">
          <property name="placeholderText">
           <string>expiry date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_inf">
          <property name="text">
           <string>&amp;No expire</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="4">
       <widget class="UnitInput" name="edit_unit">
        <property name="placeholderText">
         <string>unit</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_name">
        <property name="text">
         <string>Spec</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_unit">
        <property name="text">
         <string>Unit</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_price">
        <property name="text">
         <string>Price</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="spin_qty">
        <property name="maximum">
         <double>999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_expiry">
        <property name="text">
         <string>Expiry</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_qty">
        <property name="text">
         <string>Quantity</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_book">
        <property name="text">
         <string>Booking</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="DateInput" name="edit_book">
        <property name="placeholderText">
         <string>booking time</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="combo_products">
        <property name="insertPolicy">
         <enum>QComboBox::NoInsert</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_products">
        <property name="text">
         <string>Product</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="PriceInput" name="edit_price">
          <property name="placeholderText">
           <string>price</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_uprice">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>&amp;Unit price</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="4">
       <widget class="TextInput" name="edit_spec">
        <property name="placeholderText">
         <string>specificator</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="TextInput" name="edit_invoice">
        <property name="placeholderText">
         <string>invoice number</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="4">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="action_addnext">
       <property name="text">
        <string>&amp;Insert</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="action_addexit">
       <property name="text">
        <string>Insert and &amp;Exit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="action_cancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DateInput</class>
   <extends>QLineEdit</extends>
   <header>DateInput.h</header>
  </customwidget>
  <customwidget>
   <class>TextInput</class>
   <extends>QLineEdit</extends>
   <header>TextInput.h</header>
  </customwidget>
  <customwidget>
   <class>PriceInput</class>
   <extends>QLineEdit</extends>
   <header>PriceInput.h</header>
  </customwidget>
  <customwidget>
   <class>UnitInput</class>
   <extends>QLineEdit</extends>
   <header>UnitInput.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>combo_products</tabstop>
  <tabstop>edit_spec</tabstop>
  <tabstop>spin_qty</tabstop>
  <tabstop>edit_unit</tabstop>
  <tabstop>edit_price</tabstop>
  <tabstop>check_uprice</tabstop>
  <tabstop>edit_invoice</tabstop>
  <tabstop>edit_book</tabstop>
  <tabstop>edit_expiry</tabstop>
  <tabstop>check_inf</tabstop>
  <tabstop>action_addnext</tabstop>
  <tabstop>action_addexit</tabstop>
  <tabstop>action_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
